src/BuildingBlocks/EventBus/EventBusServiceBus/EventBusServiceBus.cs

```diff
@@ -126,17 +126,18 @@ public void UnsubscribeDynamic<TH>(string eventName)
         public void Dispose()
```

Comment:
Where is this called from? IEventBus does not implement IDisposable and I don't see this called directly.

Reply:
Really strange, it seems like never been called. I check the origin file in branch master, there is 0 references for this method too.